### PR TITLE
feat: nix package

### PR DESCRIPTION
**Pull request by tulilirockz**
This PR adds a nix package (and some fixes for it). It would be really awesome to distribute this project as a nix package because it could be bundled into many kinds of packages (including docker, rpm, deb...) and it would provide a reproducible environment for anyone that wants to run this project on their machine (provided they have nix).

### CODE REVIEW

File: flake.nix [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+            paths = if dependencies != null then [(wrapScript name)] ++ dependencies else [(wrapScript name)];
+            buildInputs = [pkgs.makeWrapper];
+            postBuild = if name == pname then "wrapProgram $out/bin/${name} --prefix PATH : $out/bin" else "wrapProgram $out/bin/${name} --prefix PATH : $out/bin/${pname}-${name}";
+          };
+      in {
+        packages = {
+          default = self.packages.${system}.blincus;
+          ${pname} = generateRunnablePackage { 
+            name = "${pname}";
+            dependencies = (with pkgs; [incus packer jq xorg.xhost coreutils gnugrep gnused getent util-linux dconf jq coreutils-full]);
```
Comment (bketelsen) [inline flake.nix:34]: jq on here twice

File: src/initialize.sh [hunk 2]
```diff
@@ -8,6 +8,9 @@ if ! test -f "${CONFIG_FILE:-}"; then
   echo "--> Config file created at ${CONFIG_FILE}."
 fi
 
-sanity
+
+if [ "$BLINCUS_SKIP_SANITY" != 1 ] ; then
```
Comment (bketelsen) [inline src/initialize.sh:12]: good idea

File: flake.nix (hunk 1 above)
Comment (tulilirockz) [reply flake.nix:34]: Oooh yeah ok sorry fixing it

**Comment by bketelsen** [comment]
I had to extract packer_build into a function because this solution didn't keep arguments when called from buildall. Thanks for the pull request!

**Pull request merged**